app/views/ErrorTemplateNoauth.scala.html

```diff
@@ -16,18 +16,19 @@
 
 @import scala.concurrent.ExecutionContext
 
-@this(mainTemplate: MainTemplate)
+@this(mainTemplate: MainTempleteNew)
 
 @(pageTitle: String, heading: String, message1: String, additionalMessages: List[String])(implicit request: Request[_],
         messages: Messages, templateRenderer: uk.gov.hmrc.renderer.TemplateRenderer, ec: ExecutionContext)
 
 @mainTemplate(
     title=pageTitle,
-    hidePtaAccountNav = true
+    backLinkContent = None,
+    showPtaAccountNav = false
 ){
-    <div class="section">
-        <h1 class="h1-heading">@heading</h1>
-        <div class="grid grid-2-3">
+    <div class="govuk-grid-row">
+        <h1 class="govuk-heading-l">@heading</h1>
+        <div class="govuk-grid-column-two-thirds">
```

Comment:
31 and 30 need to be swapped around, the formatting doesnt look quite right. So like
`<div class="govuk-grid-row">
        <div class="govuk-grid-column-two-thirds">
            <h1 class="govuk-heading-l">@heading</h1>`

Reply:
Ahh okay, good spot, I will amend when home